### PR TITLE
[ENHANCEMENT] align recently dash view table

### PR DESCRIPTION
**Pull request by nicolastakashi**
# Description

Align recently dash view table with left side table.

# Screenshots

## Before
image

## After

image

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).

## UI Changes

- [x] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.
- [x] Code follows the [UI guidelines](https://github.com/perses/perses/blob/main/ui/ui-guidelines.md).
- [x] Visual tests are stable and unlikely to be flaky. See [Storybook](https://github.com/perses/perses/tree/main/ui/storybook#visual-tests) and [e2e](https://github.com/perses/perses/tree/main/ui/e2e#visual-tests) docs for more details. Common issues include:
  - Is the data inconsistent? You need to mock API requests.
  - Does the time change? You need to use consistent time values or mock time utilities.
  - Does it have loading states? You need to wait for loading to complete.

### CODE REVIEW

**Review by AntoineThebaud — APPROVED** [review]
LGTM, I wonder if there's a better solution to avoid this magic number of 3% for top margin, anyway that's good enough for the time being.

**Review by Nexucis — APPROVED** [review]
Nice ! So cool you have the pixel perfect in the eyes !!

**Comment by nicolastakashi** [comment]
> LGTM, I wonder if there's a better solution to avoid this magic number of 3% for top margin, anyway that's good enough for the time being imho.

Me too!
There're two magical number which I'd like to avoid, but I don't have a better idea :(

File: ui/app/src/views/projects/RecentlyViewedDashboards.tsx [hunk 1]
```diff
@@ -27,14 +27,23 @@ export function RecentlyViewedDashboards(props: RecentlyViewedDashboardsProps) {
...
+        }}
+      >
         <Stack direction="row" alignItems="center" gap={1} my={2}>
           <HistoryIcon />
           <Typography variant="h3">Recently Viewed Dashboards</Typography>
         </Stack>
       </Stack>
       <ErrorBoundary FallbackComponent={ErrorAlert}>
-        <Card>
+        <Card sx={{ marginTop: '3%' }}>
```
Comment (Gladorme) [inline ui/app/src/views/projects/RecentlyViewedDashboards.tsx:46]:
```suggestion
        <Card sx={{ marginTop: 2 }}>
```

File: ui/app/src/views/projects/RecentlyViewedDashboards.tsx [hunk 2]
```diff
@@ -27,14 +27,23 @@ export function RecentlyViewedDashboards(props: RecentlyViewedDashboardsProps) {
 
   return (
     <Box id={props.id}>
-      <Stack direction="row" alignItems="center" justifyContent="space-between">
+      <Stack
+        sx={{
+          borderBottom: 1,
+          borderColor: 'divider',
+          width: '100%',
+          height: '72px',
```
Comment (nicolastakashi) [inline ui/app/src/views/projects/RecentlyViewedDashboards.tsx:35]: @Gladorme any idea about this one as well 😄 ?

File: ui/app/src/views/projects/RecentlyViewedDashboards.tsx (hunk 1 above)
Comment (nicolastakashi) [reply ui/app/src/views/projects/RecentlyViewedDashboards.tsx:46]: Cool!
Let me check this one.

File: ui/app/src/views/projects/RecentlyViewedDashboards.tsx (hunk 2 above)
Comment (Gladorme) [reply ui/app/src/views/projects/RecentlyViewedDashboards.tsx:35]: I was checking, but I think your solution is good 😄

**Review by Gladorme — APPROVED** [review]
Ahahaha, in my last PR I did something similar, but yours is better with cards aligned ([PR](https://github.com/perses/perses/commit/8b1f5d3fda84b81d9d55a99c8f71c80bfb4fd842#diff-ca6a9b6c8dd59b2125916c048699dd29ffa6fee2b8f802bbf1ef74ae61981404R224)) 🤫

LGTM! 😄

**Pull request merged**